services/analytics/api/src/main/kotlin/io/element/android/services/analytics/api/AnalyticsStore.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2025 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+ * Please see LICENSE files in the repository root for full details.
+ */
+
+package io.element.android.services.analytics.api
+
+import kotlinx.coroutines.flow.Flow
+
+/**
+ * Local storage for:
+ * - user consent (Boolean);
+ * - did ask user consent (Boolean);
+ * - analytics Id (String).
+ */
+interface AnalyticsStore {
```

Comment:
I do not understand why this interface seems to be added by this PR. It seems that it already removed from `DefaultAnalyticsStore.kt`, but not from this PR.

Reply:
Ah, I now see this wasn't needed, I could have achieved the same with `AnalyticsService.didAskUserConsent()` 🫤 , I missed that one.

Reply:
yes, the store must stay internal. But I do not see the removing of previous `AnalyticsStore` in the diff. Do you see it on your side?

Reply:
It's been moved in https://github.com/element-hq/element-x-android/commit/3597774dd245d6dac7c86d0d533ed970c4645926

Reply:
OK, there is 2 interfaces `AnalyticsStore` :).
There need to be moved back to the `impl` module.